nuget/lib/BlingFireUtils.cs

```diff
@@ -1,26 +1,40 @@
+// <copyright file="BlingFireUtils.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+//     Licensed under the MIT License.
+// </copyright>
+
 using System;
 using System.Collections.Generic;
-using System.Text;
 using System.Runtime.InteropServices;
+using System.Text;
 
 /// <summary>
 /// This is C# interface for blingfiretokdll.dll
 /// </summary>
 namespace BlingFire
 {
-    public static class BlingFireUtils
+    public static class BlingFire
     {
         private const string BlingFireTokDllName = "blingfiretokdll";
 
         [DllImport(BlingFireTokDllName)]
-        public static extern Int32 GetBlingFireTokVersion();
+        static extern Int32 GetBlingFireTokVersion();
+
+        public static Int32 GetBlingFireTokVersion(int fake = 0)
+        {
+            return GetBlingFireTokVersion();
+        }

```

Comment:
I am not sure this code works... did you run it? it looks like an infinite recursive call. Is there a way in C# to specify exactly which function to call? Other option is to just keep it simple and bite the bullet and add a public function GetVersion() which calls  private GetBlingFireTokVersion().

Reply:
reverted this change

Reply:
I feel it is best to not make this change at this moment.